packages/front-end/pages/settings/index.tsx

```diff
@@ -916,64 +916,6 @@ const GeneralSettingsPage = (): React.ReactElement => {
                       </div>
```

Comment:
```
export const DEFAULT_SRM_THRESHOLD = 0.001;
```
probably belongs in `packages/shared/src/constants.ts`

Reply:
I considered this, but SRM checks are only done in the front-end right now